content/docs/command-reference/fetch.md

```diff
@@ -228,53 +228,51 @@ the cache entries for the `data/features` directory, and
 ## Example: With dependencies
 
 After following the previous example (**Specific stages**), only the files
-associated with the `prepare.dvc` stage file have been fetched. Several
+associated with the `prepare` stage has been fetched. Several
 dependencies/outputs of other pipeline stages are still missing from the cache:
 
 ```dvc
 $ dvc status -c
 ...
-    deleted:            model.pkl
     deleted:            data/features/test.pkl
     deleted:            data/features/train.pkl
-    deleted:            data/data.xml
```

Comment:
Are you sure `data/data.xml` isn't printed here?

Reply:
Yes. I double checked this. Initially I thought it isn't printed because `data.xml` is already because of first example. But trying this example itself from starting gave same output.

Reply:
> it isn't printed because data.xml is already because of first example

Makes sense.